### DISCUSSION TITLE
Velociraptor and SOv2

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5422] [Unsupported Versions] [answered]
Currently having an issue with Velociraptor connecting back to SOv2.3.70.

All firewalls are open and amended for both windows and SO. Any suggestions on how to resolve connection issue?
![20210907_043143](https://user-images.githubusercontent.com/70167373/132312629-dee10505-6ea0-4bcc-af32-3f9069cec07b.jpg)

**Comment by weslambert** [comment, marked as answer]
Hi @ColeVan, please post your issue to the following repo: https://github.com/weslambert/SOARLab. Thanks!
